### PR TITLE
Bugfix france parser price calc

### PR DESCRIPTION
**Pull request by steinwinde**
Replaced "hours" by "hour" to make the FR.py work.

### CODE REVIEW

**Comment by ajoga** [comment]
@steinwinde you also pushed in this PR the commits you PR'd via #2450. Could you remove them from this PR please ?

File: parsers/FR.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def fetch_price(zone_key, session=None, target_datetime=None,
                 if value.text == 'ND':
                     continue
                 period = int(value.attrib['periode'])
-                datetime = start_date.replace(hours=+period).datetime
+                datetime = start_date.replace(hour=+period).datetime
```
Comment (jarek) [inline parsers/FR.py:174]:
```suggestion
                datetime = start_date.shift(hours=+period).datetime
```

see https://github.com/tmrowco/electricitymap-contrib/issues/2416#issuecomment-619422705

**Comment by steinwinde** [comment]
@ajoga @jarek - thanks!
I've also removed the reference to an issue that I thought I fix, but fail to do so.

**Comment by robertahunt** [comment]
Just want to get this included in the upcoming bump of contrib :)

**Pull request merged**